.github/workflows/tiledb-cloud-py.yaml

```diff
@@ -233,8 +233,7 @@ jobs:
 
       - name: Build wheel
         run: |
-          pip install --upgrade pip build wheel setuptools setuptools-scm
-          pip install -r requirements-py3.9.txt
```

Comment:
Build dependencies are defined in pyproject.toml and will be fetched by our build backend. The test dependencies aren't needed at build time.